pykechain/client.py

```diff
@@ -1105,6 +1105,106 @@ def create_activity(
             parent._cached_children.append(new_activity)
         return new_activity
 
+    def clone_activities(
+        self,
+        activities: List[Union[Activity2, Text]],
+        parent: Union[Activity2, Text],
+        activity_update_dicts: Optional[Dict] = None,
+        clone_parts: Optional[bool] = False,
+        clone_part_instances: Optional[bool] = True,
+        clone_children: Optional[bool] = True,
+        excluded_parts: Optional[List[Text]] = None,
+        part_parent_model: Optional[Union[Part2, Text]] = None,
+        part_parent_instance: Optional[Union[Part2, Text]] = None,
+        part_model_rename_template: Optional[Text] = None,
+        part_instance_rename_template: Optional[Text] = None,
+        asynchronous: Optional[bool] = False,
+        **kwargs
+    ) -> List[Activity2]:
+        """
+        Clone multiple activities.
+
```

Comment:
```suggestion

    .. versionadded:: 3.7
        The bulk clone activities with parts API is included in KE-chain backend since version 3.6.

```